mage_ai/api/operations/base.py

```diff
@@ -171,7 +171,7 @@ async def __create_or_index(self):
     async def __delete_show_or_update(self):
         updated_options = await self.__updated_options()
         res = await self.__resource_class().process_member(
-            self.pk, self.user, **updated_options)
+            self.pk, self.user, action=self.action, **updated_options)
```

Comment:
Could you check whether you can use `api_operation_action` in the udpated_options? https://github.com/mage-ai/mage-ai/blob/master/mage_ai/api/operations/base.py#L298

Reply:
Yes, that kwarg does work. Updated.